commands/stats.js

```diff
@@ -10,28 +10,28 @@ const getLooks = () => {
 };
 
 const getMMR = () => {
-  return rngService.normalDistribution(5000, 5000 / 3);
+  return rngService.normalDistribution(5000, 5000 / 3, 1, true);
 };
 
 const getSalary = () => {
   return rngService.logNormalDistribution(1, 1, 50000);
 };
 
 const getLength = () => {
-  const randomInches = rngService.normalDistribution(6, 6 / 3);
-  return `${randomInches} in (${Math.floor(randomInches * 2.54)} cm)`;
+  const randomInches = rngService.normalDistribution(6, 6 / 3, 1, true);
+  return `${randomInches} in (${Math.round(randomInches * 2.54)} cm)`;
 };
 
 const getHeight = () => {
-  const randomInches = rngService.normalDistribution(67, 67 / 3);
+  const randomInches = rngService.normalDistribution(67, 67 / 5, 1, true);
   return `${Math.floor(randomInches / 12)}'${Math.floor(
     randomInches % 12
-  )}" (${Math.floor(randomInches * 2.54)} cm)`;
+  )}" (${Math.round(randomInches * 2.54)} cm)`;
 };
 
 const getWeight = () => {
-  const randomPounds = rngService.normalDistribution(175, 175 / 3);
-  return `${randomPounds} lb (${Math.floor(randomPounds / 2.205)} kg)`;
+  const randomPounds = rngService.normalDistribution(175, 175 / 5, 1, true);
```

Comment:
did you intend to change the STDEV here? I know you're making further adjustments to weight/height.. but just checking.

Reply:
yes, this is mentioned in the pull request's description